### PR TITLE
In addition to #7

### PR DESCRIPTION
**Pull request by kuraga**

### CODE REVIEW

File: minify.sh [hunk 1]
```diff
@@ -8,14 +11,18 @@ fi
...
+
 dest=${1:-gentoo-build}
-mkdir $dest
 if ! sed --version 2>&1 | grep -q 'GNU sed'; then
   SEDCMD=gsed
 else
   SEDCMD=sed
 fi
+
+mkdir $dest
```
Comment (OndrejHome) [inline minify.sh]: Is there a reason for moving  the `mkdir` here? (If there is no reason, then lets keep it the old way)

**Review by OndrejHome — CHANGES_REQUESTED** [review]
Hi @kuraga , Thank you for another set of changes, one questions to this one about mkdir in code.

File: minify.sh (hunk 1 above)
Comment (kuraga) [reply minify.sh]: No, just separate "variables set" phase. 
Feel free to change

Comment (kuraga) [reply minify.sh]: @OndrejHome , done.

**Comment by OndrejHome** [comment]
THank you @kuraga !

**Pull request merged**